templates/CRM/Admin/Form/Setting/Miscellaneous.tpl

```diff
@@ -73,9 +73,26 @@
             </td>
           </tr>
         {/foreach}
-
+        <tr class="crm-miscellaneous-form-block-recentItemsMaxCount">
```

Comment:
Nothing at all, this was a merge conflict I ran into and thought I'd cleared out of the PR I submitted. (Those lines are shown in [this PR's final patch](https://patch-diff.githubusercontent.com/raw/civicrm/civicrm-core/pull/8923.patch) but not changed.)